### DISCUSSION TITLE
Add NC, DT, DC to the game player (now playing)

### DISCUSSION THREAD

**Discussion by DarkGradient** [ppy/osu#26085] [Ideas]
It may seem useless, but some people sometimes like to listen to music in the game...
I would like it to be possible to choose the mode of how to play: accelerated + pitch (NC), slow (DC), fast without pitch (DT).

**Comment by DarkGradient** [comment]
Well, looks like no one want to add this useful feature.

**Comment by Gabixel** [comment]
They're human too, and can miss some threads. Bumping may help, but there is really no need to get mad.
I must admit that I did not fully understand your suggestion. Are you referring to... the already usable mods, or something else?
You're too vague, what do you mean by those combinations? I would like to see a detailed example.

Also, keep in mind that lazer (or osu! in general) is not intended to be a music player, for specific reasons (but I'm just reporting what I've often seen developers say in other threads, no need to discuss it here)

**Reply by Gabixel** [reply]
Oh, you're referring to add something like some global "dedicated pitch/speed presets" in the audio player on the toolbar? Maybe that's it, right?

Again, it's probably going to be rejected because the music player has limited features for a reason, I think.
Aaand.. you can just use mods (and custom presets) for that, which is way more customizable if you think about it.
They're also against keeping mods modifiers in the main menu (i.e. speed and pitch rate) from what I've noticed.
